pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java

```diff
@@ -1890,6 +1890,41 @@ public void testCompilationInvokedFunction() {
     decoded = and.getOperands().get(1).getFunctionCall().getOperands().get(1).getLiteral().getStringValue();
     Assert.assertEquals(encoded, "key1%3Dvalue+1%26key2%3Dvalue%40%21%242%26key3%3Dvalue%253");
     Assert.assertEquals(decoded, "key1=value 1&key2=value@!$2&key3=value%3");
+
+    query = "select toBase64('hello!'), fromBase64('aGVsbG8h') from mytable";
```

Comment:
Can you add tests for nested transform as well ?

Reply:
- toBase64 is inside another transform function that produces string
- toBase64 is inside fromBase64
- etc ...

Reply:
Done - added nested function calls with other transform functions(eg. upper, reverse) both in SELECT and in filter.